src/frontend/components/UpdatableVideoPlayer/UpdatableVideoPlayer.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+import { defineMessages, FormattedMessage } from 'react-intl';
+import styled from 'styled-components';
+
+import { Video } from 'types/Video';
+import { Button } from '../Button/Button';
+import { LayoutMainArea } from '../LayoutMainArea/LayoutMainArea';
+import { ROUTE as FORM_ROUTE } from '../VideoForm/VideoForm';
+import { VideoPlayer } from '../VideoPlayer/VideoPlayer';
+import { withLink } from '../withLink/withLink';
+
+interface UpdatableVideoPlayerProps {
+  video: Video;
+}
+
+export const ROUTE = () => '/player-updatable';
+
+const messages = defineMessages({
+  button: {
+    defaultMessage: 'Update',
+    description: 'Send instructor on video form page to upload a new video',
+    id: 'components.updateVideo.button',
+  },
+});
+
+const UpdateVideoContainer = styled(LayoutMainArea)`
```

Comment:
If you look at #138, we don't want the video + instructor specific controls to fit into the main area. We actually want the instructor controls flowing out of the main area.